packages/server/package.json

```diff
@@ -7,6 +7,7 @@
     "author": "Hedera Smart Contracts Team",
     "dependencies": {
         "@hashgraph/json-rpc-relay": "file:../relay",
+        "@kasa/koa-request-id": "^2.0.1",
```

Comment:
were there other options?
Doesn't seem heavily utilized or maintained.
Guess the package itself is simple and it seems to only `require('uuid')` so it's safe

Reply:
"safe" - now, yes, but isn't this the kind of dependency that has caused, for other applications, unnecessary trauma at later dates?  For such a small thing?  Can't we take advantage of the MIT license and just clone it into a repo that we control? Or just rewrite it, or include it?  MIT license!?  (Basically I think Nana is right to question this kind of thing.)

Reply:
Agree with both Nana and David, doesn't seem necessary to use a whole package for a id generator. IMO, it would be better to implement our own solution. We can use the `koa-request-id` as reference, if needed.

Reply:
Applied requestId functionality to the server as a middleware.